### ISSUE TITLE
Are COG Range Requests Supported

### ISSUE THREAD

**Issue by efreethy** [danwild/leaflet-geotiff-2#44]
I have a large Geotiff (500 MB). Its cloud optimized in COG format. Can this library handle that? Most likely through range requests to get only the necessary bytes out of the file.

**Comment by anton-seaice** [comment]
Not really, see link, it grabs the whole image (at the highest resolution if there are overviews)

```
      const image = await this.tiff.getImage(this.options.image).catch((e) => {
```

https://github.com/onaci/leaflet-geotiff-2/blob/d954d78ae69c79dd53d89014d7850e0e7a8d4a1a/src/leaflet-geotiff.js#LL190C10-L190C10
